samtranslator/model/sam_resources.py

```diff
@@ -593,6 +593,12 @@ def _validate_dlq(self):
                 "'DeadLetterQueue' requires Type and TargetArn properties to be specified.".format(valid_dlq_types),
             )
 
+        if not (isinstance(self.DeadLetterQueue.get("Type"), str)):
```

Comment:
can you update this method to validate that the `DeadLetterQueue` is of type Dict, and if not raise a validation error.

Reply:
I don't know this is necessary as this is already handled by [sam](https://github.com/aws/serverless-application-model/blob/develop/samtranslator/model/sam_resources.py/#L68). And this error is thrown when `DeadLetterQueue` is not dict `Type of property 'DeadLetterQueue' is invalid.`

Reply:
not sure where I can find this error, but I found this usage where we just check that the `self.DeadLetterQueue` is not None 
https://github.com/aws/serverless-application-model/blob/cae272936f714999fb962d97fa6a8b68507b14b3/samtranslator/model/sam_resources.py#L139-L140